src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -130,8 +130,8 @@ Specific iEEG fields SHOULD be present:
 | DCOffsetCorrection              | RECOMMENDED           | [string][]                           | A description of the method (if any) used to correct for a DC offset. If the method used was subtracting the mean value for each channel, use "mean".                                                                                                                                                                                  |
 | HardwareFilters                 | RECOMMENDED           | [object][] of [objects][] or `"n/a"` | [Object][] of temporal hardware filters applied, or `"n/a"` if the data is not available. Each key:value pair in the JSON object is a name of the filter and an object in which its parameters are defined as key:value pairs. For example, `{"Highpass RC filter": {"Half amplitude cutoff (Hz)": 0.0159, "Roll-off": "6dB/Octave"}}` |
-| ElectrodeManufacturer           | RECOMMENDED           | [string][]                           | Can be used if all electrodes are of the same manufacturer (for example, AD-TECH, DIXI). If electrodes of different manufacturers are used, please use the corresponding table in the \_electrodes.tsv file.                                                                                                                           |
-| ElectrodeManufacturersModelName | RECOMMENDED           | [string][]                           | If different electrode types are used, please use the corresponding table in the \_electrodes.tsv file.                                                                                                                                                                                                                                |
+| ElectrodeManufacturer           | RECOMMENDED           | [string][]                           | Can be used if all electrodes are of the same manufacturer (for example, `"AD-TECH"`, `"DIXI"`). If electrodes of different manufacturers are used, please use the corresponding table in the \_electrodes.tsv file.                                                                                                                           |
+| ElectrodeManufacturersModelName | RECOMMENDED           | [string][]                           | If different electrode types are used, please use the corresponding table in the `\_electrodes.tsv` file.                                                                                                                                                                                                                                |
```

Comment:
The backslash was there just to escape the underscore, before this was not formatted as a codeblock prior to your change

```suggestion
| ElectrodeManufacturersModelName | RECOMMENDED           | [string][]                           | If different electrode types are used, please use the corresponding table in the `*_electrodes.tsv` file.                                                                                                                                                                                                                                |
```

Reply:
Ah yes!!!

I ddi think "why is that there?!" when scanning through that file.